pages.zh/common/java.md

```diff
@@ -0,0 +1,24 @@
+# java
+
+> Java 程序启动器.
+> 更多信息: <https://java.com>.
+
+- 通过提供类名称运行一个含有 main 函数的 java .class 程序:
+
+`java {{classname}}`
+
+- 运行一个 .jar 程序:
+
+`java -jar {{filename.jar}}`
```

Comment:
You can translate the term `filename` if you want.

Reply:
I think I will keep it as is. Translating just `filename` while leaving `.jar` looks a bit funny. Thanks for the suggestion thou!